### PR TITLE
Feature/464 resume's skills cannot be deleted

### PR DESCRIPTION
**Pull request by alex-flores-jalafund**
[AB#464](https://dev.azure.com/funjaladev/DevLevel3/_sprints/backlog/DevLevel3%20Team/DevLevel3/Sprint%208%20-%20Final?workitem=464)

# Overview

image

### CODE REVIEW

File: src/components/resumes/ResumesAPI.js [hunk 1]
```diff
@@ -87,6 +87,7 @@ export async function getSkillsByName(name) {
 }
 
 export async function postSkillsById(idUser, Skill) {
+  console.log(idUser, Skill);
```
Comment (ela-ocana-jalafund) [inline src/components/resumes/ResumesAPI.js]: delete this console.log :)

File: src/components/resumes/Box.jsx [hunk 2]
```diff
@@ -39,6 +39,7 @@ const useStyles = makeStyles((theme) => ({
 
 function Box(props) {
   const { id, contact, person, summary, skills } = props;
+  console.log(skills);
```
Comment (ela-ocana-jalafund) [inline src/components/resumes/Box.jsx]: Delete this console.log :)

File: src/components/resumes/ResumesAPI.js (hunk 1 above)
Comment (alex-flores-jalafund) [reply src/components/resumes/ResumesAPI.js]: it's done

File: src/components/resumes/Box.jsx (hunk 2 above)
Comment (alex-flores-jalafund) [reply src/components/resumes/Box.jsx]: it's done

**Review by ela-ocana-jalafund — APPROVED** [review]

**Review by paola-quintanilla-jalafund — APPROVED** [review]

**Review by lina-ramirez-jalafund — APPROVED** [review]

**Pull request merged**